tests/gold_tests/continuations/double_h2.test.py

```diff
@@ -109,9 +109,13 @@
 
 def make_done_stat_ready(tsenv):
     def done_stat_ready(process, hasRunFor, **kw):
-        retval = subprocess.run("traffic_ctl metric get continuations_verify.test.done > done  2> /dev/null", shell=True, env=tsenv)
+        done_file = os.path.join(process.RunDirectory, "continuations_verify.done")
+        retval = subprocess.run(
```

Comment:
Or how about getting rid of the redirection/grep and doing it in pure python?

eg
```
   retval = subprocess.run("traffic_ctl metric get continuations_verify.test.done", shell=True, capture_output=True, text=True, env=tsenv)
       return retval.returncode == 0 and '1' in retval.stdout
```

Reply:
Yeah, that's a good idea. Let me try that.